### PR TITLE
Jkralik/fix/cloud registering

### PR DESCRIPTION
**Pull request by jkralik**

### CODE REVIEW

File: api/cloud/oc_cloud_store.c [hunk 1]
```diff
@@ -32,6 +32,18 @@ check oc_config.h and make sure OC_STORAGE is defined if OC_CLOUD is defined.
 #endif
 
 #define CLOUD_STORE_NAME "cloud"
+#define ci_server ci_server
```
Comment (Danielius1922) [inline api/cloud/oc_cloud_store.c:35]: Capital letters? That's usually the convention for macros.

**Review by Danielius1922 — APPROVED** [review]

File: api/cloud/oc_cloud_store.c (hunk 1 above)
Comment (jkralik) [reply api/cloud/oc_cloud_store.c]: Fixed

File: api/cloud/oc_cloud_internal.h [hunk 2]
```diff
@@ -148,6 +148,7 @@ void cloud_manager_stop(oc_cloud_context_t *ctx);
 
 void oc_create_cloudconf_resource(size_t device);
 
+bool cloud_is_permanent_access_token(int64_t expires_in);
```
[the remark this answers is not in the document]
Comment (jkralik) [reply api/cloud/oc_cloud_internal.h:156]: Fixed

File: api/cloud/oc_cloud_manager.c [hunk 3]
```diff
@@ -156,6 +161,10 @@ static int
 _register_handler(oc_cloud_context_t *ctx, oc_client_response_t *data)
 {
   oc_cps_t cps = OC_CPS_FAILED;
+  char *access_token = NULL, *refresh_token = NULL, *uid = NULL;
```
[the remark this answers is not in the document]
Comment (jkralik) [reply api/cloud/oc_cloud_manager.c]: Fixed

**Comment by WAvdBeek** [comment]
Josef, do we still need this one?
because all work has been done on the Jakarta branch (which is now merged)

**Comment by jkralik** [comment]
@WAvdBeek Yes, we need it because without it the CTT 1.4.1, 1.4.5, and 1.4.8 will be never passed.

**Review by ondrejtomcik — APPROVED** [review]

**Pull request merged**